src/Page/Guide/View.elm

```diff
@@ -356,4 +356,11 @@ printButtonStyle =
         ]
 
 
+outerVideoContainerStyle : Style
+outerVideoContainerStyle =
+    batch
```

Comment:
this can go in without the batch wrapper since it's one style but also OK as it is.